README.md

```diff
@@ -1,88 +1,87 @@
 # doc.silverstripe.org
 
-This is the source code powering http://docs.silverstripe.org. It
-primarily consists of the SilverStripe
+This repository contains the source code powering SilverStripe's developer documentation website https://docs.silverstripe.org.
+
+The source code here consists primarily of the SilverStripe
 [framework](https://github.com/silverstripe/silverstripe-framework)
 and the [docsviewer](https://github.com/silverstripe/silverstripe-docsviewer)
 module with minimal configuration.
 
-For adding functionality or editing the style of the documentation see the 
-[docsviewer](http://github.com/silverstripe/silverstripe-docsviewer) module.
-
-## Development
-
-To set up a test instance:
-
- * Clone this repository to a LAMP server.
- * Install [Composer](http://docs.silverstripe.org/en/getting_started/composer)
- * Install [sake](https://docs.silverstripe.org/en/developer_guides/cli/).
- * After installing composer run `composer install --prefer-source` to grab the modules.
- * Run the docs crontask in the browser `dev/tasks/UpdateDocsCronTask`
-   to download all fresh markdown documentation files and reindex them. Note: this
-   will take some time to run. Alternatively, you can use sake
-   to perform these tasks by firstly running the command `sake
-   dev/tasks/RefreshMarkdownTask flush=1` and secondly `sake
-   dev/tasks/RebuildLuceneDocsIndex flush=1`.
- * Make sure to flush the cache for markdown content to show up.
-
-## Source Documentation Files
```

Comment:
We need to keep this bit @SpiritLevel so others know how to add new documentation source / versions in the future.